src/envoy/utils/filter_state_utils.h

```diff
@@ -26,6 +26,8 @@ constexpr char kOperation[] =
     "com.google.espv2.filters.http.path_matcher.operation";
 constexpr char kQueryParams[] =
     "com.google.espv2.filters.http.path_matcher.query_params";
+constexpr char kApiKey[] =
```

Comment:
can we prefix these names with FilterState  or change the name space to filter_state?

Reply:
Agreed, we should change the naming before API Gateway team depends on it. Is there an envoy convention for naming filter state?